### PR TITLE
create checkout page with order overview

### PR DESCRIPTION
**Pull request by oy-gen**
including all price calculations && shipping conditions

### CODE REVIEW

File: components/CheckoutTable.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+          <th align="right">
+            {TotalTaxes.toLocaleString('de-DE', {
+              style: 'currency',
+              currency: 'EUR',
+            })}
+          </th>
+        </tr>
+        <tr>
+          <td empty />
+          <td empty />
```
Comment (Chaosfuchs) [inline components/CheckoutTable.js:92]: Do you really need two of this? If it's so, I said nothing :)

File: components/CheckoutTable.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    </>
+  );
+}
+
+const StyledTable = styled.table`
+  font-size: 0.9rem;
+  padding: 1rem 0;
+  color: var(--text-maincolor);
+  width: 100%;
+  border-spacing: 0;
```
Comment (Chaosfuchs) [inline components/CheckoutTable.js:111]: If the border-spacing is 0, I think you don't need this :)

File: components/CheckoutTable.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+  tr:nth-child(even) {
+    background-color: white;
+  }
+
+  tr:last-of-type th {
+    font-size: 1rem;
+    font-weight: 600;
+    border-top: 2px solid var(--text-maincolor);
+    border-bottom: 0;
```
Comment (Chaosfuchs) [inline components/CheckoutTable.js:125]: Here also, like the border spacing :)

File: components/CheckoutTable.js [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  td {
+    padding: 0.4rem;
+    line-height: 1.2rem;
+    border-bottom: 1px solid var(--text-lightcolor);
+  }
+
+  td:empty {
+    border: 0;
```
Comment (Chaosfuchs) [inline components/CheckoutTable.js:143]: Border 0 does nothing, or does this remove the Tableborders?

**Review by Chaosfuchs — COMMENTED** [review]
This looks really awesome :) You should think about, whether you want to make a button or link on the checkout page that leads back to the product page :)

File: components/CheckoutTable.js (hunk 1 above)
Comment (oy-gen) [reply components/CheckoutTable.js:92]: maybe there is another solution for that. They serve as a placeholder. If I remove them, the following tds are moved on their position, which I don't want.

File: components/CheckoutTable.js (hunk 2 above)
Comment (oy-gen) [reply components/CheckoutTable.js:111]: it overwrites the default border-spacing, which is 2px.

File: components/CheckoutTable.js (hunk 4 above)
Comment (oy-gen) [reply components/CheckoutTable.js:143]: it overwrites previous styling in this special case

File: components/CheckoutTable.js (hunk 3 above)
Comment (oy-gen) [reply components/CheckoutTable.js:125]: it overwrites previous styling in this special case

**Pull request merged**